changelog/fragments/1716490302-Load-fleet.ssl.certificate_authorities-from-agent-policy.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: Load fleet.ssl.certificate_authorities from agent policy
```

Comment:
Could we rephrase this in more end-user-friendly terms?

Reply:
Reworded in f43e4f3320cd983d9ddee4ab0afa0e2e0683f083